pelicun/assessment.py

```diff
@@ -305,3 +335,1438 @@ def scale_factor(self, unit: str | None) -> float:
             scale_factor = 1.0
 
         return scale_factor
+
+
+class Assessment(AssessmentBase):
```

Comment:
Right, here things were a bit uncertain. It would be great if we could have a discussion on how to better manage these classes. The reason we have both `Assessment` and `DLCalculationAssessment` there currently is because I was initially planning to only have an `Assessment` class written from scratch, but then realized it's more straightforward to refactor `DL_calculation.py` by moving out code and placing it in `assessment.py`. I would recommend polishing `DLCalculationAssessment` a bit, eventually renaming it, and removing `Assessment`.

Reply:
Agreed, that works.